README.md

```diff
@@ -125,23 +119,17 @@ A list of companies that have paid Developer Community Writer Programs.
   > Technical content and tutorials about DevSecOps, Cloud security, Kubernetes security.
 
 - [Magic](https://magic-fortmatic.typeform.com/to/Wgzsocor) - Up to $300 per piece
-  [Link to Magic Website](https://magic.link/)
-  > Technical tutorials on how to use Magic
+  > Technical tutorials on how to use Magic.link
 
 - [Make Use Of](https://www.makeuseof.com/contributor/) - $120 per piece with performance benefits
   > Tutorials and features about consumer apps and software products.
 
-- [Mixster](https://mixstersite.wordpress.com/2019/05/24/mixster/#more-2253)
-
-- [Nanonet](https://nanonets.com/blog/write-for-us/)
-  > Get paid to write about your favourite machine learning topics!
+- [Nanonet](https://nanonets.com/blog/write-for-us/) - Pay unknown
+  > Get paid to write about your favorite machine learning topics.
 
-- [Neptune](https://neptune.ai/write-for-us) - Up to $500 per article.
+- [Neptune](https://neptune.ai/write-for-us) - $300-$600 per article.
   > Technical articles, how-to guides and tutorials on machine learning and data science.
 
-- [Okta](https://developer.okta.com/blog) - Paid through Toptal based on your hourly rate
-  > Technical tutorials and demos using Okta's products.
```

Comment:
This one is confusing. You can't directly apply to work with Okta, you have to go through Toptal, so I removed this.